docs/ansible-release-announcement/minor-release/announcement-email.md

```diff
@@ -0,0 +1,85 @@
+Release announcement : Ansible Community Package x.y.z
+
+Hello everyone,
+
+We're happy to announce the release of the Ansible x.y.z package!
+
+Ansible x.y.z requires latest version of ansible-core 2.15 and includes a curated set of
+Ansible collections that provides a vast number of modules and plugins.
```

Comment:
Can we revert to the original version, and start discussing this change once this is merged? While the new formulation is technically more correct (when considering Ansible as a pure Python package), it is inconsistent with both how the docsite and the changelog document Ansible.

(Reference: the discussion thread in https://groups.google.com/g/ansible-project/c/bAP_VSgXBiI, and quite a few discussions of this kind in the past, both on the mailing list and during the community meetings.)

Reply:
Is there a place where we will discuss this?

Reply:
I would guess in community-topics, and/or in a PR with a/some proposed changes. Edits should just not be mixed with the initial commit of the currently used templates. Otherwise we end up with a huge mess where it is hard to distinguish between the current versions and edits.

Reply:
Creating a separate PR.